spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskControllerTests.java

```diff
@@ -411,7 +411,7 @@ void findDslTextContainsSubstring() throws Exception {
 
 	@Test
 	void findByDslTextAndNameBadRequest() throws Exception {
-		mockMvc.perform(get("/tasks/definitions").param("dslText", "fo").param("search", "f")
+		mockMvc.perform(get("/tasks/definitions").param("dslText", "fo").param("taskName", "f")
```

Comment:
In this case we reference the param by string name rather than constant. I would be in favor of moving the `TASK_NAME_REQUEST_PARAMETER` to a static final constant or just using `"taskName"` everywhere.